pkg/preflight/preflight.go

```diff
@@ -160,13 +160,27 @@ func Run(appID string, appSlug string, sequence int64, isAirgap bool, archiveDir
 		preflight.Spec.Collectors = collectors
 
 		go func() {
-			logger.Debug("preflight checks beginning")
+			logger.Info("preflight checks beginning")
 			uploadPreflightResults, err := execute(appID, sequence, preflight, ignoreRBAC)
 			if err != nil {
 				logger.Error(errors.Wrap(err, "failed to run preflight checks"))
 				return
 			}
-			logger.Debug("preflight checks completed")
+
+			// Log the preflight results if there are any warnings or errors
+			// The app may not get installed so we need to see this info for debugging
+			if GetPreflightState(uploadPreflightResults) != "pass" {
+				// TODO: Are there conditions when the application gets installed?
```

Comment:
I don't think so here since we should only deploy if the preflight state is `"pass"` https://github.com/replicatedhq/kots/blob/main/pkg/preflight/preflight.go#L236-L245